policy/pipeline/required_tasks.rego

```diff
@@ -76,7 +127,7 @@ deny contains result if {
 }
```

Comment:
I think this should not fail if pipeline-required-tasks is provided and has the right pipeline runtime key in it.

Reply:
Which rule shoudn't fail?

Reply:
`missing_required_data`

GH wouldn't let me click further up :sob: